.changeset/cold-items-notice.md

```diff
@@ -0,0 +1,7 @@
+---
+'@siafoundation/explored-js': minor
+'@siafoundation/explored-react': minor
+'@siafoundation/explored-types': minor
+---
+
+Added the TransactionChainIndices route types, sdk method, and react hook.
```

Comment:
Since this changeset will get published as a change in each of the included libraries, either write it more generically so the whole description applies to all of them, or if that is not accurate do individual ones. That way explored-js changelog doesn't mention react etc. eg of something that can describe this change for all 3: 
```
Added the TransactionChainIndices API.
```